src/app/portlets/dot-edit-page/content/dot-edit-content.component.html

```diff
@@ -7,8 +7,11 @@
     class="dot-edit__toolbar"
     #toolbar>
 </dot-edit-page-toolbar>
-
-<div class="dot-edit__page-wrapper">
+<dot-edit-content-view-as-toolbar
+    (changeViewAs)="changeViewAsHandler($event)"
```

Comment:
I think it's cleaner if we handle just one change output here and you toolbar should be smart enough to send just the data that change and in you handler you check what data came and do the action accordingly.